### ISSUE TITLE
`mpm upgrade` requires `--all` parameter for full upgrade since v5.4.0

### ISSUE THREAD

**Issue by mnetzwur** [kdeldycke/meta-package-manager#715]
Since updating to version 5.4.0, mpm upgrade on macOS fails with "critical: No package provided for upgrade." 

OS: macOS 12.3.1
mpm: 5.4.0

Expected behavior: upgrade packages as before.

**Comment by kdeldycke** [comment]
Oh yes that's my fault.

In `5.4.0` I introduced a way for the `upgrade` operation to accept one or more parameters to update a couple of packages in one go. So now you can do:
```shell-session
$ mpm upgrade firefox
$ mpm upgrade sqlite mysql postgresql
```

To make it more obvious the new `upgrade` support package IDs as parameters, I choose to require one:
```shell-session
$ mpm upgrade
critical: No package provided for upgrade.
```

This is the problem you have right?

Up to `5.3.x` a simple call to `mpm upgrade` would upgrade all packages from all managers. Now to replicate the same behavior you need to add the`--all` parameter:
```shell-session
$ mpm upgrade --all
Upgrade all outdated packages from brew...
Upgrade all outdated packages from cask...
(...)
```

What do you think of this design decision? Do you think reverting to `mpm upgrade` as an alias to `mpm upgrade --all` is safe?

**Comment by kdeldycke** [comment]
Thinking about this, I ended up concluding that indeed, having `--all` option assumed on a bare `upgrade` call is a great quality of life. I pushed a fix upstream and it will be available in the upcoming v5.5.0 release.
